webapp/components/navbar_dropdown.jsx

```diff
@@ -312,7 +309,7 @@ export default class NavbarDropdown extends React.Component {
                                 />
                             </a>
                         </li>
-                        {adminDivider}
```

Comment:
Won't this show two dividers in a row for a non-admin with only one team on a closed server?